packaging/hudi-utilities-bundle/pom.xml

```diff
@@ -148,6 +148,7 @@
                   <include>org.apache.hive:hive-service-rpc</include>
                   <include>org.apache.hive:hive-metastore</include>
                   <include>org.apache.hive:hive-jdbc</include>
+                  <include>org.apache.hive:hive-exec</include>
 
                   <include>org.apache.hbase:hbase-client</include>
```

Comment:
Why we add the dependency to `hudi-utilities` ? I guess you mean `hudi-flink-bundle` ? There is already `hive-exec` in `hudi-flink-bundle` dependencies, the reason you can not find the class is that the hive package is shaded in release 0.10.1, while in release 0.11.0 or latest master, we have removed the shade pattern for hive, maybe you can reference the pom and have a try ~